### PR TITLE
systemctl: add is-active and is-enabled examples

### PR DESCRIPTION
**Pull request by hellojukay**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/systemctl.md [hunk 1]
```diff
@@ -25,3 +25,11 @@
 - Reload systemd, scanning for new or changed units:
 
 `systemctl daemon-reload`
+
+- Check a unit is active:
```
[the remark this answers is not in the document]
Comment (hellojukay) [reply pages/linux/systemctl.md:29]: Yes

File: pages/linux/systemctl.md [hunk 2]
```diff
@@ -25,3 +25,11 @@
...
+
+`systemctl is-active {{unit}}`
+
+- Check a unit is Enabled:
```
[the remark this answers is not in the document]
Comment (hellojukay) [reply pages/linux/systemctl.md:33]: Yes

File: pages/linux/systemctl.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/linux/systemctl.md:29]: To accept the change, click the "Commit suggestion" button in the suggested change box @hellojukay :-)

File: pages/linux/systemctl.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/linux/systemctl.md:33]: To accept the change, click the "Commit suggestion" button in the suggested change box @hellojukay :-)

[the remark this answers is not in the document]
Comment (hellojukay) [reply pages/linux/systemctl.md:33]: thanks

**Comment by navarroaxel** [comment]
Could you add this to the command description to?

```
> More information: <https://www.freedesktop.org/software/systemd/man/systemctl.html>.
```

**Comment by sbrl** [comment]
@navarroaxel: That would be a  great addition, but to avoid complicating this PR, let's leave that out for now and do it in a future PR.

**Review by sbrl — APPROVED** [review]
I've just committed the suggestion for you @hellojukay if you don't mind. I think this is ready to merge!

**Comment by hellojukay** [comment]
i fix my email.

**Pull request merged**